week-1/implement/max.js

```diff
@@ -0,0 +1,26 @@
+function findMax(arr) {
+  let maxNum = -Infinity;
+
+  /*for(let i= 0; i < arr.length; i++) {
+    const num = arr[i]; 
+ }
+    // for loop iterates through each element of an array and iterates arr.length times: It visits all elements of the array. same with for (let num of arr) {}  */
+
+  for (let num of arr) {
+    if (!isNaN(num) && num > maxNum) {
+      maxNum = num;
+    }
+  }
+  return maxNum;
+}
+
```

Comment:
I wrote my code like this: 
function getMax(array){
    return Math.max.apply(null, array);
}

module.exports = getMax;
But yours alos makes sense, didn't think of doing it like you did. Well done